plans/07-observability-devops.md

```
@@ -1309,208 +181,68 @@ jobs:
-          fi
+// After
+sourcemap: 'hidden',
 ```
 
----
+### File: `.gitignore`
 
-## Dependency Graph
+Append at the end if not already present:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)